package.json

```diff
@@ -4,12 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run unit && npm run cover && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
     "unit": "mocha --recursive test/spec/ --require test/helper --timeout 300000",
     "cover": "istanbul cover _mocha -- -R dot --recursive test/spec/ --require test/helper --timeout 300000",
-    "check-coverage": "istanbul check-coverage --statement 98.31 --branch 94.89 --function 96.30 --line 98.31",
+    "check-coverage": "istanbul check-coverage --statement 90 --branch 90 --function 90 --line 90",
```

Comment:
It doesn't hut to have it there. (AFAIK) I just thought it was an arbitrary choice.